IBPSA/Examples/Tutorial/SimpleHouse/SimpleHouse3.mo

```diff
@@ -0,0 +1,88 @@
+within IBPSA.Examples.Tutorial.SimpleHouse;
+model SimpleHouse3 "Air model"
+  extends SimpleHouse2;
+
+  parameter Modelica.Units.SI.Volume V_zone=8*8*3 "Zone volume";
+  parameter Modelica.Units.SI.MassFlowRate mAir_flow_nominal=1
+    "Nominal mass flow rate for air loop";
+  parameter Modelica.Units.SI.CoefficientOfHeatTransfer h_wall=2
+    "Convective heat transfer coefficient at the wall";
+
+  Modelica.Thermal.HeatTransfer.Components.ThermalResistor convRes(R=1/2/A_wall)
+    "Thermal resistance for convective heat transfer"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+        rotation=270,
+        origin={130,20})));
+  IBPSA.Fluid.MixingVolumes.MixingVolume zone(
+    redeclare package Medium = MediumAir,
+    V=V_zone,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    m_flow_nominal=mAir_flow_nominal)
+    "Very simple zone air model"
+    annotation (Placement(transformation(extent={{110,130},{90,150}})));
```

Comment:
Minor: Would it make sense to have zone in the wall block and name the text `Wall` `Zone` or `building/ room`? Also, the window could be added to this block. This way, all the building physics are in one block.

Reply:
I currently added the window components to the 'Wall' block. @mwetter do you think we should also move the `Zon` to this same block (and e.g. call it 'Building envelope'). Or do we leave it like it is?

Reply:
@mwetter This would be the suggested change: 
![image](https://github.com/ibpsa/modelica-ibpsa/assets/22442514/9d14d808-f3a2-4b52-a12c-919e33964b60)

The available space for cooling and ventilation is small, but I think this better reflects the actual models in the IBPSA and other libraries. 
I have it as a local commit and can push it if you agree @mwetter @jelgerjansen

Reply:
@FWuellhorst we could of course change the lay-out to something like this:
![image](https://github.com/ibpsa/modelica-ibpsa/assets/43753266/0bbedf09-92a0-4d9a-98f5-d00d455ca1c1)

which has plenty of space in all blocks. Furthermore, I centered the diagram in the 'new' lay-out (which appeared not to be centered)

Reply:
The bottom one looks great.

Reply:
I'll push my changes and also incorporate them in PR #1791